Drv/LinuxSpiDriver/LinuxSpiDriverComponentImpl.cpp

```diff
@@ -34,13 +34,26 @@ namespace Drv {
 // Handler implementations for user-defined typed input ports
 // ----------------------------------------------------------------------
 
+// @ DEPRECATED: Use SpiWriteRead port instead (same operation with a return value)
 void LinuxSpiDriverComponentImpl::SpiReadWrite_handler(const FwIndexType portNum,
                                                        Fw::Buffer& writeBuffer,
                                                        Fw::Buffer& readBuffer) {
+    FW_ASSERT(portNum >= 0, static_cast<FwAssertArgType>(portNum));
```

Comment:
Just as a note, we usually don't assert `portNum>=0` because this is done by the autocoder already, so we usually just dismiss those static analysis warnings when they pop up. Not a problem though.